params/denomination.go

```diff
@@ -32,6 +32,6 @@ package params
 //	new(big.Int).Mul(value, big.NewInt(params.GWei))
 const (
 	Wei   = 1
-	GWei  = 1e9
-	Ether = 1e18
+	GWei  = 1_000_000_000 * Wei
+	Ether = 1_000_000_000 * GWei
```

Comment:
Using scientific notation results in treating these as untyped _float_s rather than untyped _int_s. Golang will happily truncate integer constants, but it will not convert a float into an int unless it is lossless.

Reply:
Can we make our own constants instead of modifying the shared code with geth?

Reply:
moved to the `utils` package.